src/components/Board/Board.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { jest, it, beforeAll } from '@jest/globals';
+import { act, render, waitFor } from '@testing-library/react';
+import Board from './Board';
+import { setFail } from '../../api/boardsApi';
+
+jest.mock('../../api/boardsApi');
+
+beforeAll(() => {
```

Comment:
Maybe consider using `beforeEach` so you don't leave any side effects to this variable. `beforeAll` runs only once before the suite.

Reply:
done